bin/radical-pilot-control

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+__copyright__ = "Copyright 2024, http://radical.rutgers.edu"
+__license__   = "MIT"
+
+
+import os
+import sys
+import time
+
+
+# ------------------------------------------------------------------------------
+#
+def run_agent_command(sbox, cmd, args):
+
+    if not os.path.isdir(sbox):
+        raise RuntimeError('pilot sandbox not found: %s' % sbox)
+
+    import json
+    import socket
+
+    portfile = f'{sbox}/agent_0.ports.json'
+
+    if not os.path.isfile(portfile):
+        raise RuntimeError('portfile not found: %s' % portfile)
+
+    port = None
+    with open(portfile) as fin:
+        info = json.load(fin)
+        port = info.get('command')
+
+    print('command port: {%s} {%s}: %.2f' % (port, type(port), time.time()))
```

Comment:
to be removed? (otherwise it appears in task STDOUT)

Reply:
Yep, thanks!